src/DDTrace/Integrations/Laravel/LaravelSandboxedIntegration.php

```diff
@@ -0,0 +1,206 @@
+                $rootSpan->setIntegration($integration);
+                $rootSpan->setTraceAnalyticsCandidate();
+                $rootSpan->setTag(
+                    Tag::RESOURCE_NAME,
+                    $route->getActionName() . ' ' . ($route->getName() ?: 'unnamed_route')
+                );
+                $rootSpan->setTag('laravel.route.name', $route->getName());
+                $rootSpan->setTag('laravel.route.action', $route->getActionName());
+                $rootSpan->setTag('http.url', $request->url());
+                $rootSpan->setTag('http.method', $request->method());
+
+                return false;
+            }
+        );
+
+        \dd_trace_method(
+            'Illuminate\Routing\Route',
+            'run',
+            function (SpanData $span) use ($integration) {
+                $span->name = 'laravel.action';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $integration->getServiceName();
+                $span->resource = $this->uri;
+                $span->meta['integration.name'] = LaravelSandboxedIntegration::NAME;
+            }
+        );
+
+        \dd_trace_method(
+            'Symfony\Component\HttpFoundation\Response',
+            'setStatusCode',
+            function (SpanData $span, $args) use ($rootSpan) {
+                $rootSpan->setTag(Tag::HTTP_STATUS_CODE, $args[0]);
+                return false;
+            }
+        );
+
+        \dd_trace_method('Illuminate\Events\Dispatcher', 'fire', function (SpanData $span, $args) use ($integration) {
+            $span->name = 'laravel.event.handle';
+            $span->type = Type::WEB_SERVLET;
+            $span->service = $integration->getServiceName();
+            $span->resource = $args[0];
+            $span->meta['integration.name'] = LaravelSandboxedIntegration::NAME;
+        });
+
+        \dd_trace_method('Illuminate\View\View', 'render', function (SpanData $span) use ($integration) {
+            $span->name = 'laravel.view.render';
+            $span->type = Type::WEB_SERVLET;
+            $span->service = $integration->getServiceName();
+            $span->resource = $this->view;
+            $span->meta['integration.name'] = LaravelSandboxedIntegration::NAME;
+        });
+
+        \dd_trace_method(
+            'Illuminate\View\Engines\CompilerEngine',
+            'get',
+            function (SpanData $span, $args) use ($integration) {
+                $span->name = 'laravel.view';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $integration->getServiceName();
+                if (isset($args[0]) && \is_string($args[0])) {
+                    $span->resource = $args[0];
+                }
+                $span->meta['integration.name'] = LaravelSandboxedIntegration::NAME;
+            }
+        );
+
+        // Using the legacy API only for this method because of orphaned spans in sandboxed api when `return false` is
```

Comment:
I think in this case this is a very good idea, as `Illuminate\Foundation\ProviderRepository::load` could potentially be some info we want to give. E.g. if a lot of providers are provided than it could even be a source of slowness.

More in general, I would go with your suggestion if:
- it makes sense to trace a specific method
- the method is 'risky', e.g. accessing `$this` or `$args` or the returned value or the likes.
While I would vote to temporarily use `dd_trace` if tracing a specific method is pointless AND there are no risks in the callback code (in this case we are not interacting at all with laravel).